docs/data-guidelines.md

```diff
@@ -120,6 +120,27 @@ For example, the `ImageData` API has worker support, recorded like this:
 
 Formerly named `available_in_workers`, this policy was set in [#2362](https://github.com/mdn/browser-compat-data/pull/2362).
 
+### Permissions API permission descriptions (`api.Permissions.*_permission.__compat.description`)
+
+Document Permissions API permissions called `[permissionname]` as `api.Permissions.[permissionname]_permission` and add a description that reads `"<code>[permissionname]</code> permission"`
```

Comment:
Can I suggest revising this a bit to have some parallelism to the other feature name items in this doc? Something like this:

```
## Permissions API permissions (`permissionname_permission`)

Add [Permissions API](link_to_spec_or_mdn_page) permissions as subfeatures of `api.Permissions` using the name _permissionname_\_permission with the description text set to `<code>permissionname</code> permission`.
```

Also, whatever the final text of the H2, there needs to be a ToC entry at the top of the file.

Reply:
Changed to:
```
## Permissions API permissions (`permissionname_permission`)

Add [Permissions API](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API) permissions as subfeatures of [`api.Permissions`](https://developer.mozilla.org/en-US/docs/Web/API/Permissions) using the name _permissionname_\_permission with the description text set to `<code>permissionname</code> permission`.
```

Please note that I added an extra link for `api.Permissions`.

Reply:
👍

Reply:
👍 I always appreciate more links!